### PR TITLE
Sponsers and socialmedia

### PR DESCRIPTION
**Pull request by hossam-khalaf**
added fontawesome with npm,
i need help with the social media icons background,
and contact information borders

### CODE REVIEW

**Comment by hossam-khalaf** [comment]
there is a thin border between email and phone divisions , i can't handle 
thanks for your comment :)

File: dist/index.html [hunk 1]
```diff
@@ -58,11 +60,108 @@
...
+                </div>
+              </div>
+              <div class="contact-section__contact-phone">
+                <div class="phone flex">
+                  <div class="icon mt-1 mr-3 w-1/6">
+                    <img
+                      class="h-8"
+                      src="assets/img/contact/phone.png"
+                      alt="phone logo"
+                    />
```
Comment (abdalla-elmahdy) [inline dist/index.html:129]: Good work, but I think there are redundant divs here that can be simply removed and move its classes in its parent div like on line 123

File: dist/index.html [hunk 2]
```diff
@@ -58,11 +60,108 @@
...
+                  </div>
+                  <div class="w-5/6">
+                    <p>666 888 6666 Phone</p>
+                    <p class="text-xxs">Line</p>
+                  </div>
+                </div>
+              </div>
+              <div class="contact-section__contact-location">
+                <div class="phone flex">
```
Comment (abdalla-elmahdy) [inline dist/index.html:138]: I think phone class is in the wrong place here

Comment (hossam-khalaf) [reply dist/index.html:138]: yes i noticed, thankes for observation

File: dist/index.html (hunk 1 above)
Comment (hossam-khalaf) [reply dist/index.html:129]: i'm deleting the extra markup

File: dist/index.html [hunk 3]
```diff
@@ -58,11 +60,108 @@
...
 
     <section class="sponsors">
-      Sponsors
+      <!-- sponsers logos! -->
+      <div class="container">
+        <div class="flex justify-between mx-auto my-20 flex-wrap">
+          <div class="px-4 py-2 m-2">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="px-4 py-2 m-2">
```
[the remark this answers is not in the document]
Comment (hossam-khalaf) [reply dist/index.html:69]: thanks for this , i think ill make a new branch , i got many conflicts in json files, nothing seems to be worknig, again, you are such a great help

[the remark this answers is not in the document]
Comment (abdalla-elmahdy) [reply dist/index.html:69]: As you see fit, but still try solving the conflicts instead

**Pull request closed without merging**